### PR TITLE
Update/webterminal

### PR DESCRIPTION
**Pull request by shaunmulligan**

### CODE REVIEW

**Comment by shaunmulligan** [comment]
#198

File: pages/runtime/terminal.md [hunk 1]
```diff
@@ -1,50 +1,37 @@
...
 
 ## Establishing a Terminal Session
 
-Resin.io uses [tty.js][tty.js] to give you direct access to your devices. A button will appear giving you access to this interface once your device is **online** and code is **pushed to it** (if the button is not visible beneath the logging window, ensure that both of these conditions are met):-
-
-![Terminal Session Button](/img/terminal-button.png)
-
-Once a terminal session is established you'll be presented with an area in which the terminal session will be shown and a button to open the terminal:-
+  In order for you to start a terminal session in your device [container][docker-container], you first need to ensure that your device is **online** and code is **pushed to it** and is running. If your container code crashes or ends quickly, it is not possible to attach a console to it. One option to keep your containers running is to enable the INITSYSTEM in your container. This can easily be done by creating a device environment variable called `INITSYSTEM` and setting its value to `on`.
```
Comment (emirotin) [inline pages/runtime/terminal.md]: device or app-wide env var, right?

Comment (shaunmulligan) [inline pages/runtime/terminal.md]: yeah, it can be either, I had just left it as device env var because it read better

**Comment by emirotin** [comment]
LGTM

**Pull request merged**